### PR TITLE
DEV-1801: Add auth bearer token

### PR DESCRIPTION
**Pull request by fanzhao1986**
Add authbearer token in web socket client.

DEV-1801

### CODE REVIEW

File: docs/MarketData.rst [hunk 1]
```diff
@@ -0,0 +1,52 @@
+.. _marketdata:
+
+==========
+MarketData
+==========
+
+To receiving market data from reactive-platform, Use `reactive.platform.marketdata.MDClient`
```
Comment (marayl) [inline docs/MarketData.rst]: `s/receiving/receive/` and `s/Use/use/`.

File: docs/MarketData.rst [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+a API token and address. The token is issued from the trading platform under account settings:
+
+.. code:: python
+
+    from reactive.platform.marketdata.mdclient import MDClient
+
+
+    key = 'xxx'
+    url = "ws://127.0.0.1:8989"
+    md_client = Client(key=key, addr=url)
```
Comment (marayl) [inline docs/MarketData.rst]: Any chance we could swap the parameter order, so that `addr` is before `key`?

File: docs/MarketData.rst (hunk 1 above)
Comment (fanzhao1986) [reply docs/MarketData.rst]: my bad. fixed.

File: example/marketdata.py [hunk 3]
```diff
@@ -15,16 +15,32 @@
...
 
 MARKET_LIST = ["EURUSD-REX", "EURGBP-REX", "EURCHF-REX", "EURRON-REX", "USDCAD-REX",
-               "USDCHF-REX", "USDJPY-REX", "USDCHF-REX", "EURJPY-REX", "AUDUSD-REX"]
+               "USDCHF-REX", "USDJPY-REX", "USDCHF-REX", "EURJPY-REX", "AUDUSD-REX",
+               "BCHUSD-BFN", "BTCUSD-BFN", "ETHUSD-BFN", "LTCUSD-BFN", "XRPUSD-BFN",
+               "BCHUSD-BIN", "BTCUSD-BIN", "ETHUSD-BIN", "LTCUSD-BIN", "XRPUSD-BIN",
+               "BCHUSD-CNB", "BTCUSD-CNB", "ETHUSD-CNB", "LTCUSD-CNB", "XRPUSD-CNB"]
+
+# this TOKEN can be used for connecting mdgw with default config.
+TOKEN = "eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzUxMiIsImtpZCI6InVVajh3bVV5b0NkZFkyRU95U1E4Vkc4SEVHZlFNSE" \
```
Comment (markmcdowell) [inline example/marketdata.py]: Remove token from code.

File: example/marketdata.py [hunk 4]
```diff
@@ -44,19 +60,19 @@ def md_print_handler(md: MarketData):
...
         print("unsub", market)
         await c.unsubscribe([market])
-        await asyncio.sleep(2)
+        await asyncio.sleep(1)
 
 
 if __name__ == "__main__":
     # use default setting
-    client = MDClient(addr='ws://127.0.0.1:8989/md')
+    client = MDClient(addr='ws://127.0.0.1:8989', key=TOKEN)
```
Comment (markmcdowell) [inline example/marketdata.py]: Remove local host from example, it should specify the platform url.

File: src/reactive/platform/client.py [hunk 5]
```diff
@@ -37,22 +37,25 @@ class Client(metaclass=Cached):
     Client represents client object to access reactive exchange platform.
     """
 
-    def __init__(self, key=None, url="https://api.crossfire-dev.reactivemarkets.net"):
+    URL = "https://api.crossfire-dev.reactivemarkets.net"
```
Comment (markmcdowell) [inline src/reactive/platform/client.py]: Don’t expose our dev url

**Comment by markmcdowell** [comment]
Please remove all instances of the dev url, remove local host and port examples. We should only use the public urls.

File: example/marketdata.py (hunk 4 above)
Comment (fanzhao1986) [reply example/marketdata.py]: I specify one, but not sure if it is the right one we will use.

**Comment by marayl** [comment]
@fanzhao1986 @markmcdowell is this PR ready to merge?

**Comment by markaylett** [comment]
@fanzhao1986 this has been open for over 2 weeks now.

**Comment by markmcdowell** [comment]
The urls are still wrong, do we have an endpoint for the new md websocket yet?
Similar to api.platform.reactivemarkets.com

**Comment by markaylett** [comment]
> The urls are still wrong, do we have an endpoint for the new md websocket yet?
> Similar to api.platform.reactivemarkets.com

Should be, but I need to test. I will get this done by tomorrow.

**Comment by fanzhao1986** [comment]
> The urls are still wrong, do we have an endpoint for the new md websocket yet?
> Similar to api.platform.reactivemarkets.com

Change all url in the example or docs to `api.platform.reactivemarkets.com`

**Pull request merged**